src/child/Child.ts

```diff
@@ -53,6 +53,10 @@ export class Child extends ChildMessanger{
         this.send(MessageType.UrlChanged, url);
     }
 
+    public participantLeftMeeting() {
```

Comment:
why not closeLiveEvent?
for example, seller dashboard sends "setList" and not technologyChanged

Reply:
done